### ISSUE TITLE
Add transparency and details to Getting Started documentation

### ISSUE THREAD

**Issue by tonesoutherland** [IHE/supplement-template#29]
**The Problem**
The [Getting Started guide](https://github.com/IHE/supplement-template/wiki/Getting-Started) is helpful, good information is provided. However, there is a strong dependency on one person: John. John's name is mentioned several times in the guide. There is some magic that John does to make things happen at certain steps. What happens if John disappears (not that he is planning to)? How do the other team members know what to do in those particular steps.

**Proposed Solution**
Replace "John" with a designated role, assign John to that role (these are just words, so easy enough to do, should this move forward). 

More importantly, describe in detail what happens for certain steps that are "black box" such as this one: 

_John will setup the webhook to auto-build when you push updates https://us-central1-fhir-org-starter-project.cloudfunctions.net/ig-commit-trigger._

How is a webhook setup to auto-build? Is there a dependency on an external system or organizations (i.e HL7?). What are the steps involved here?

The goals is to make this process transparent and repeating so it's easy for anyone to pick up the torch and run with it.

**Comment by costateixeira** [comment]
I agree and while I could add the name "José" and others in most of those
steps, there should be consistency which at this moment is ensured by John.
We should have transparency and ability to scale without asking for John to
profusely document every reason and dependency he knows, but to automate
the procedures and simplify handover:

We can automate a lot as we've done for Belgium and OpenHIE to allow things
to scale.

- automation to create a GitHub repo (there's a python script, and even
better, a template.
- automation to setup the webhooks (same python script or a GitHub action
that calls the hl7 webhook which is in the template)
- automatic validation of the IG to see that the necessary data is there
(we have some options there)


While there should be a manual gateway, there's no need to depend on a
specific person to do all this. I think someone can and should keep track
of what is being created but the manual labour can be taken care of.

In practice, perhaps the solution is to add an opaque layer: A committee
member asks for the creation of a repo from (a role). The rest is handled
for the user.

If it's of interest, we can discuss options in a call so that we have a
joint plan with the best opportunities. The important thing would be to
have a plan for these longer-term aspects (that are meanwhile available).
